### PR TITLE
changed titel to topics and added pictures

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/swisscom/multisense-general/life_sign_event.schema.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "$id": "https://akenza.io/swisscom/multisense-life_sign_event.schema.json",
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "title": "Swisscom Multisense",
+  "title": "Life sign event",
```
Comment (mariusgiger) [inline types/swisscom/multisense-general/life_sign_event.schema.json:4]: should this be lifecylce?

File: types/xovis/xovis-PC2S/meta.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
   "description": "People counting and tracking",
   "author": "Akenza AG",
   "firmwareVersion": "V1.0.0",
+  "loraDeviceClass": "N/A",
```
Comment (mariusgiger) [inline types/xovis/xovis-PC2S/meta.json:9]: remove it if it is not relevant

**Review by mariusgiger — CHANGES_REQUESTED** [review]
almost good

File: types/swisscom/multisense-general/life_sign_event.schema.json (hunk 1 above)
Comment (J2thatsme) [reply types/swisscom/multisense-general/life_sign_event.schema.json:4]: yes

**Review by mariusgiger — APPROVED** [review]
please check the tests

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**